src/main/resources/swagger/api-docs.yaml

```diff
@@ -7380,7 +7381,7 @@ paths:
           description: |
             An identifier of the tool version for this particular tool registry, for example `v1`.
 
-              *In FireCloud, this must be an integer representing the FireCloud snapshot id.*
```

Comment:
nit: Terra snapshot id sounds bit confusing- could this be more clear/specific?

Reply:
`method snapshot id` might be more accurate, I will think about this a bit more but change it to that if I can't think of anything